tensorflow/python/kernel_tests/array_ops/pad_op_test.py

```diff
@@ -279,7 +279,8 @@ def testIntTypes(self):
 
   def testFloatTypes(self):
     self.skipTest("b/183965033")
```

Comment:
These tests are currently being skipped. If I comment out the skip, they all pass except for the empty tensor case ` self._testAll(np.random.rand(0, 3, 4).astype(t), [[0, 0], [2, 1], [2, 3]], 0.0)`.

Is the test currently being skipped due to the issue with the empty tensor case? If so, I can move that test to its own function and disable it so that we can re-enable the float tests.

Reply:
I don't know for sure since the internal bug has almost no info, but when I run it fails due to timing out after 1 minute. The case with shape `(12, 13, 14)` takes too much time, since I think the gradient checker does work proportional to the number of output elements. Lowing the size to `(1, 3, 4)` and padding amounts to `[[0, 0], [1, 1], [2, 2]]` for that test and disabling the empty tensor test causes it to pass.

Can you lowering the shape size for the test with shape `(12, 13, 14)`, and delete the test with the empty shape (or move it to a separate disabled test method if you prefer)? You can do so in this PR. Then it should pass.

Reply:
Thanks for the review!

Thanks for the suggestions, I have just pushed the fixes. That test was taking ~!0 minutes on my machine.